### DISCUSSION TITLE
Only allow users to manage their own content

### DISCUSSION THREAD

**Discussion by michaelguntenaar** [directus/directus#3519] [Archive]
We would like to give guestbloggers their own login but we want them to have the ability to only create items and see/edit the items they have created. Could not find it in the documentation for the new version of Directus (the old ones showed it).

********
Nevermind got confused by the "old" documentation where the "owner" field was mentioned.

Adding

```json
{
    "user_created": {
        "_eq": "$CURRENT_USER"
    }
}
```

did the job as expected ;-)

**Comment by benhaynes** [comment]
Adding the author's answer here:

```
{
    "user_created": {
        "_eq": "$CURRENT_USER"
    }
}
```

**Reply by afonsolopez** [reply]
Not working for me on 9.25.1 :(
